services/skus/service_nonint_test.go

```diff
@@ -837,6 +837,40 @@ func TestCreateOrderWithReceipt(t *testing.T) {
 				},
 			},
 		},
+
+		{
+			name: "successful_case_android_vpn_monnthly",
```

Comment:
looks like you have a typo here

Reply:
Good spotting, thanks, fixed in https://github.com/brave-intl/bat-go/pull/2426.